include/vcpkg/packagespec.h

```diff
@@ -112,10 +110,11 @@ namespace vcpkg
         InternalFeatureSet features;
 
         FullPackageSpec() = default;
-        explicit FullPackageSpec(PackageSpec spec, InternalFeatureSet features)
+        FullPackageSpec(PackageSpec spec, InternalFeatureSet features)
```

Comment:
Why did you drop `explicit` here? I think the other one should be `explicit` too....

Reply:
Because we are not a library, I don't think we need to be hyper-defensive about explicit. We should still use `explicit` for single-argument constructors to avoid accidental implicit conversions.